backend/clickhouse/cursor.go

```diff
@@ -47,18 +47,18 @@ func getConnection[T interface{}](edges []*Edge[T], pagination Pagination) *Conn
 		}
 	} else if pagination.After != nil && len(*pagination.After) > 1 {
 		hasPreviousPage = true // implicitly true because the passed in cursor should match
-		if len(edges) == LogsLimit+1 {
+		if len(edges) >= LogsLimit+1 {
 			hasNextPage = true
 			edges = edges[:len(edges)-1]
 		}
 	} else if pagination.Before != nil && len(*pagination.Before) > 1 {
 		hasNextPage = true // implicitly true because the passed in cursor should match
-		if len(edges) == LogsLimit+1 {
+		if len(edges) >= LogsLimit+1 {
 			hasPreviousPage = true
 			edges = edges[1 : len(edges)-1]
```

Comment:
Was also wondering about this line and if we shouldn't be trimming on the front and end of the `edges` 🤔 

```suggestion
			edges = edges[1:]
```

Reply:
Not going to change this for now. Doesn't actually seem to be causing issues in the tests I was looking at.